skillmap/src/App.tsx

```diff
@@ -427,6 +433,10 @@ class AppImpl extends React.Component<AppProps, AppState> {
             }
         }
 
+        if (!this.props.signedIn || (this.props.signedIn && this.state.cloudSyncCheckHasFinished)) {
+            this.setState({ syncingLocalState: false });
```

Comment:
Curious why this needs to happen on every state change.

Reply:
oh i just added it in the spirit of overkill because i was afraid of having the new loader permanently hide the skillmap in weird edge cases; probably setting it anywhere we set cloudSyncCheckStateHasFinished to true would be enough though

Reply:
"spirit of overkill" :) :). I think it will trigger a redraw every time (i could be wrong!). Maybe add a condition to check for `this.state.syncingLocalState == true`? Or trust that it's always getting set correctly in the cloud sync check.

Reply:
ah you're right! i thought setstate was smart about setting the same value but it definitely redraws in this case. will update!

Reply:
oh, huh... i also thought setstate was smart

Reply:
only for functional components it seems

Reply:
I'm loving the trend in github profile icons happening here :)

Reply:
well you're a trendsetter!